pagopa/io-app
# feat: [IOCOM-642] Handle of ONGOING status on SEND's payments

## Short description
This PR adds support for the "ongoing" payment status on SEND multiple payments
| |
|-|
|![Simulator Screenshot - iPhone 14 - 2023-10-30 at 17 43 58](https://github.com/pagopa/io-app/assets/5150343/0b94a5bd-12ea-4323-abba-8de831109a65)|

## List of changes proposed in this pull request
- DS updated to 1.15.x
- Handling of the new in-progress status for both badge and text

## How to test
Using the _io-dev-api-server_, generate a SEND payment whose failure is `PAA_PAGAMENTO_IN_CORSO ` and check that the UI reports it as "In Corso" / "Ongoing"
